src/peers/gov/nasa/jpf/vm/JPF_java_lang_Class.java

```diff
@@ -643,12 +643,12 @@ int getField (MJIEnv env, int clsRef, int nameRef, boolean isRecursiveLookup) {
   
   @MJI
   public int getDeclaredField__Ljava_lang_String_2__Ljava_lang_reflect_Field_2 (MJIEnv env, int clsRef, int nameRef) {
-    return getField(env,clsRef,nameRef, false);
+    return getField(env,clsRef,nameRef, false, false);
```

Comment:
Could you please add a comment explaining why parameter `publicOnly` is set to `false` here?